### ISSUE TITLE
Dionea nymph additions?

### ISSUE THREAD

**Issue by BlueSushii** [Apollo-Community/ApolloStation#615]
While I was playing as a nymph I thought of a few cool things it should be able to do.
I would love to see the ability for people to add a headset to the nymphs or maybe a id slot. (maybe make a sub category human with just a lot of things disabled. make it so you could dress them up as Ian.
I just want to see this in-game so more people would be them. they are cute little creatures and should be seen more often and I think these changes might change that.

in summary. the ability to give a id or radio headset to a nymph, maybe even a hat?

**Comment by runedog48** [comment]
- [ ] ID Slot
- [ ] Radio Slot
- [ ] Equipment ( Hats, etc )

**Comment by lmas** [comment]
Moved to https://apollo-community.org/viewtopic.php?f=10&t=674
